### PR TITLE
LPS-69144 Add test

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by leoadb** [comment]
ci:reopen

File: modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarResourceServiceTest.java [hunk 1]
```diff
@@ -60,16 +68,42 @@ public void setUp() throws Exception {
 	}
 
 	@Test
-	public void testSearchCount() throws Exception {
+	public void testAddResourceWithModelPermission() throws Exception {
```
Comment (marcellustavares) [inline modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarResourceServiceTest.java]: @leoadb, is this testing the described bug? can we rename this to something else to describe the effect of your fix?

`addResourceWithModelPermission` is not very clear.

Thx

**Comment by marcellustavares** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcellustavares** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/44918. [See changes here.](https://github.com/marcellustavares/liferay-portal/compare/3721708~1...3721708) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**